mobile/src/pages/game/game.ts

```diff
@@ -42,21 +42,22 @@ export class GamePage {
     return this.goals[player.id];
   }
 
-  playerGoal(playerId){
-    if(!this.isFinish()){
-      let player = _.find(this.players, {id: playerId});
-      this.goals[player.id] += 1;
-      this.goalsHistory[player.team].push(player.id);
-      this.score[player.team] = this.score[player.team] + 1;
-    }
+
+  playerGoal(playerId, own:boolean = false){
+    if (this.isFinish()) return;
+
+    const player = _.find(this.players, {id: playerId});
+    this.goals[player.id] = own ? --this.goals[player.id] : ++this.goals[player.id];
+    this.setGoalsOnTeam(player, own);
   }
 
-  reduceGoal(team){
```

Comment:
I don't know why you removed reduce goal? I think we should keep it because somebody can add goal by mistake.